en/components/general/update-guide.md

```diff
@@ -51,6 +51,168 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 14.1.x to 14.2.x
+### General
+- Ignite UI for Angular now has a peer dependency on [igniteui-theming](https://github.com/IgniteUI/igniteui-theming). Install the theming package and add the following preprocessor configuration in your `angular.json` file.
+
+    ```sh
+    npm install igniteui-theming
+    ```
+
+    ```json
+        "build": {
+          "options": {
+            "stylePreprocessorOptions": {
+                "includePaths": ["node_modules"]
```

Comment:
couldn't this be done in a migration?

Reply:
We don't have migration instructions to do that.

Reply:
Well, I remember that in our update to Angular@12 our `angular.json` was updated by Angular.
So, I figured the same should be possible for us, unless there's some kind of protection for it.
In that case it's more a question of if it's worth the effort or not.

Reply:
Can't tell you if it's worth or not, but I don't think we have time to implement it before the release.